### ISSUE TITLE
Enhancement: Post instructions for Base64 file validity check

### ISSUE THREAD

**Issue by roleerob** [crypto-com/chain-desktop-wallet#838]
**Is your feature request related to a problem? Please describe.**

Yes, the problem is I do not know how to carry out the instructions listed.

**Describe the solution you'd like**

Here is what we read:

> *"When verifying the SHA512 checksum please do use the Base64 encoding instead of the default hex encoding to match the output we provide in Base64"*

We are given no instructions on exactly how to use Base64 encoding.  I have always used the Windows utility, certutil.  Which reports its SHA512 result in Hex I guess, since it doesn't match your number.  In checking out certutil's help details, I could not find anything related to Base64.  I make no profession to being an expert, so maybe it is somewhere obvious, but I (and others?) would find it very helpful for you to give us the details of how to carry this Base64 validity check out.

**Describe alternatives you've considered**

I do not know of any alternative.  I simply installed the latest version without checking its validity, which is, of course, not ideal ...

**Additional context**

**Comment by frederikstroem** [comment]
I do agree that it is not standard practice to distribute checksums this way, but their reasoning behind this descision is [mentioned](https://github.com/crypto-com/chain-desktop-wallet/issues/463#issuecomment-828186184) in #463. You can also take a look at #370 and #380.

I also hope they would consider switching to normal checksums. Adding PGP signatures as well would be an A+ in my book. 😊

**Comment by frederikstroem** [comment]
On my Linux machine, I was able to turn the SHA512 command output into a base64 encoded string using the following command.

```
sha512sum <filename> | cut -d " " -f 1  | xxd -r -p -g 0 | base64 -w 0
```

Where `<filename>` is replaced with the filename of the file to be verified. So for instance when I checked the latest AppImage, I used the following command line sequence.

```
❯ sha512sum chain-desktop-wallet-0.6.3-x86_64.AppImage | cut -d " " -f 1  | xxd -r -p -g 0 | base64 -w 0
Px0Ju/FwBq+i6AOx6537KjI7BK6zpnwElUqbdoacMiT5B1XpP7TQKbHrqVv5XShrGE1BQpTw5Z2sn1BwIqc2TA==
```

The output of this command is in exactly the same format as distributed in the release page.

**Comment by roleerob** [comment]
I use Windows 10 Professional and have always relied upon its CERTUTIL utility, e.g.:

certutil -hashfile chain-desktop-wallet-setup-0.6.7.exe SHA512

With your response, I looked into them all and found this utility referenced:

[Hex to Base64 Converter](https://base64.guru/converter/encode/hex)

With it, I was able to convert the Hex results of the command line CERTUTIL referenced above to the Base64 value displayed on the GitHub website.  A bit of a headache to have to take this much time for this extra step, but ...  It works! At least for now, until someone decides it is worthwhile to supply what I have requested (i.e. they conduct this extra step themselves ... 🧐 ... as a ONE time "customer service" to the rest of us (MANY, MANY times) ... 😉)

I will not "hold my breath" on this latter point, so ...  Will close this issue.

Thank you for the assist @frederikstroem!
